Web Forum/appsettings.json

```diff
@@ -7,6 +7,6 @@
   },
   "AllowedHosts": "*",
   "ConnectionStrings": {
-    "DefaultConnection": "Server=localhost;Port=3306;Database=web_forum;User=root;Password=root;"
+    "DefaultConnection": "Server=localhost;Port=3306;Database=web_forum;User=root;Password=password123;"
```

Comment:
Nice password XD

Reply:
Yeah.. I decided to leave it because if anyone would repeat it would be me... but still I ashamed :|